x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/index.tsx

```diff
@@ -40,19 +39,15 @@ const allRulesTabs = [
     id: AllRulesTabs.rules,
     name: i18n.RULES_TAB,
     disabled: false,
+    href: '/rules',
```

Comment:
should we use our constants?

Reply:
I think we can remove href, it is not in used.